saleor/graphql/account/tests/test_account.py

```diff
@@ -4511,6 +4511,10 @@ def test_query_customers_with_sort(
         ({"search": "wroc"}, 1),  # city
         ({"search": "pl"}, 1),  # country
         ({"search": "+48713988102"}, 1),
+        ({"search": "Alice Kowalski"}, 1),
+        ({"search": "Kowalski Alice"}, 1),
+        ({"search": "John Doe"}, 1),
+        ({"search": "Alice Doe"}, 0),
```

Comment:
`Q(email__ilike=search_value)` I think it's missing in parametrize.

Reply:
There is, by `mirumee.com"` as a first test.